repository_service_tuf/helpers/api_client.py

```diff
@@ -89,11 +91,10 @@ def is_logged(settings: LazySettings):
 
 
 def get_headers(settings: LazySettings) -> Dict[str, str]:
```

Comment:
I can't comment exactly on the affected lines of this function, but messages on lines 103 (Try re-login: **'rstuf --auth admin login'**) and 109 (The token has expired. Run **'rstuf --auth admin login'**) are not 100% accurate anymore, since you don't necessarily need to login but you just might have an invalid token.

Reply:
I removed the token from this PR. I will propose it separately.
I notice that we need to make the `get_headers` more clear.